### ISSUE TITLE
Question about the calculation of padding_percentage.

### ISSUE THREAD

**Issue by Bo396543018** [CarperAI/trlx#448]
The calculation of padding_percentage in the file modeling_ppo.py is n / mask.numel(). This seems to be the effective number of tokens. Am I understanding this correctly?

**Comment by reciprocated** [comment]
Yes, you are! This was logging the effective number of tokens (inversely related to the padding percentage)
